src/components/Toaster/Toaster.tsx

```diff
@@ -139,20 +91,21 @@ export const toast = ({
         actions ? { important: true } : {} // focused if actionable
     );
 
+const bemToaster = bem('Toaster', styles);
+
 export const Toaster = ({ duration = 2500, ...props }) => (
     <SonnerToaster
+        {...bemToaster.block()}
```

Comment:
This can lead to a bug. Basically, this thing will set the prop `className`. The way the package works is that it takes the props passed and calculates what the classes should be. If the props include a `classname` already, it will add that too, to allow users to add additional styles if needed. So with the current code we can have 2 bugs:
1. Props aren't passed to `block()` so if classname is defined it will not be included in the applied styles
2. `props` are spread after `block` is spread. if `classname` is defined in the `props` it will overwrite the classname added by `block`
So we need it to be last in the spread, and called with `props` as an argument